en/getting_started/basic_flight_modes.md

```diff
@@ -1,186 +1,179 @@
 
+![MC Manual Flight](../../images/flight_modes/manual_stabilized_MC.png)
 
-## COPTER FLIGHT MODES
-
-## Manual Modes
+#### Acro (Acrobatic Mode)
 
-Manual modes are with the least assistance from the PX4 autopilot or even without assistance to the pilot. 
+For copters this is the 3D flying mode. You can make flips and acrobatics. In acrobatic mode you have fully direct rate control over your aircraft. You have to remember that once you release Pitch and Roll to their center position the craft remains in its current state. You have to give back command in order to level it or change course and direction. In Acrobatic mode you have full rate control over the Roll, Pitch, Yaw and Ttrottle.
 
-**MANUAL - STABILIZED FLIGHT**
+![MC Manual Acrobatic Flight](../../images/flight_modes/manual_acrobatic_MC.png)
 
-In Stabilized flight your Copter will hover in level flight once the Pitch and Roll sticks are at center position. You will have a full manual angle control over the multicopter. If your craft is perfectly set and balanced it will hold the altitude if Throttle is at 50% in the middle of the scale at calm weather. If there is a wind blowing the craft will drift in the direction of the wind and you have to control the throttle all the time during wind gusts to hold altitude. In order to perform a maneuver, you have to hold your Pitch, Roll and Yaw command, otherwise the craft will level itself the moment you release the control sticks.
+#### Rattitude
 
-![MC Manual Flight](../../images/flight_modes/manual_stabilized_MC.png)
- 
+In simple terms it is the mix between Manual/Stabilized and Acro mode. When the Roll/Pitch stick is centered or if you move it up to halfway then the craft behaves like in Manual/Stabalized mode. If you move Roll/Pitch stick beyound halfway to full command the the copter behaves like in Acro mode. This way you could fly in the comfort of stabalized flight but still be able to perform flips and tricks with your copter.
  
-**ACROBATIC**
 
-For multicopter and helicopters it is the 3D flying mode. You can make flips and acrobatics. In acrobatic mode you have fully direct rate control over your aircraft. You have to remember that once you release Pitch and Roll to their center position the craft remains in its current state. You have to give back command in order to level it or change course and direction. In Acrobatic you have full rate control over the Roll, Pitch, Yaw and Throttle.
+### Assisted Modes
 
-![MC Manual Acrobatic Flight](../../images/flight_modes/manual_acrobatic_MC.png)
- 
-**RATITUDE**
+Assisted modes are with more assistance to the pilot from the autopilot. Assisted modes need manual pilot input for the roll/pitch/yaw but automatically control the thrust to hold a fixed altitude at centered throttle stick. Vertical speed is controlled by manual input from the pilot to throttle stick but in a rate of descend and ascend fashion. The horizontal speed is controlled by the pitch/roll stick input.
```

Comment:
> Vertical speed is controlled by manual input from the pilot to throttle stick but in a rate of descend and ascend fashion. 

This could be phrased as:

Vertical speed/rate of ascend and descend is controlled by throttle stick input.